src/extension/commandPaletteHandler.ts

```diff
@@ -96,7 +96,8 @@ export class CommandPaletteHandler {
 
         return this.reactNativePackager.start()
             .then(() => {
-                return new CommandExecutor(this.workspaceRoot).spawnReactCommand(command, args, null);
+                return new CommandExecutor(this.workspaceRoot).spawnReactCommand(command,
```

Comment:
No. I just verified that Command Palette: Run-Android didn't work before and it seems to be working now.